src/modules/loadPopup.js

```diff
@@ -0,0 +1,180 @@
+    const popupReservationContentlist = document.createElement('ul');
+    popupReservationArray.forEach((reservation) => {
+      const reservationItem = document.createElement('li');
+
+      const dateStart = document.createElement('span');
+      dateStart.innerText = reservation.date_start;
+      const dateEnd = document.createElement('span');
+      dateEnd.innerText = reservation.date_end;
+      const userName = document.createElement('span');
+      userName.innerText = reservation.username;
+
+      reservationItem.append(dateStart, ' - ', dateEnd, ' by ', userName);
+      popupReservationContentlist.append(reservationItem);
+    });
+    popupReservationContent.append(popupReservationContentlist);
+  }
+
+  // RESERVATIONS FORM
+
+  const popupFormReservation = document.createElement('form');
+  popupFormReservation.method = 'post';
+  popupFormReservation.onsubmit = handleReservationSubmit;
+
+  const reservationFormHead = document.createElement('h2');
+  reservationFormHead.innerText = 'Add A Reservation';
+  const popupReservationInput1 = document.createElement('input');
+  popupReservationInput1.name = 'date_start';
+  popupReservationInput1.placeholder = 'Start Date';
+  popupReservationInput1.type = 'date';
+  const popupReservationInput2 = document.createElement('input');
+  popupReservationInput2.name = 'date_end';
+  popupReservationInput2.placeholder = 'End Date';
+  popupReservationInput2.type = 'date';
+  const popupReservationInput3 = document.createElement('input');
+  popupReservationInput3.name = 'username';
+  popupReservationInput3.placeholder = 'User Name';
+  const popupReservationSubmit = document.createElement('button');
+  popupReservationSubmit.type = 'submit';
+  popupReservationSubmit.innerText = 'Submit';
+
+  popupComment.append(popupCommentHeader, popupCommentContent);
+  popupFormComment.append(popupInput1, popupInput2, popupSubmit);
+  popupCommentContainer.append(popupComment, popupFormComment);
+  popupReservation.append(popupReservationHeader, popupReservationContent);
+  popupFormReservation.append(
+    popupReservationInput1,
+    popupReservationInput2,
+    popupReservationInput3,
+    popupReservationSubmit,
+  );
+  popupReservationContainer.append(popupReservation, popupFormReservation);
+
+  if (button.className === 'comment') {
+    // If comment popup is clicked
+    commentPopup.append(popupClose, popupimg, popupTitle, popupdes, popupCommentContainer);
+    popupContC.classList.toggle('active');
+    popupClose.addEventListener('click', () => {
+      commentPopup.innerHTML = '';
+      popupContC.classList.toggle('active');
+    });
+  } else {
+    // If reservations popup is clicked
+    reservationsPopup.append(popupClose, popupimg, popupTitle, popupdes, popupReservationContainer);
+    popupContR.classList.toggle('active');
+    popupClose.addEventListener('click', () => {
+      reservationsPopup.innerHTML = '';
+      popupContR.classList.toggle('active');
+    });
+  }
+};
```

Comment:
- [ ] Good job working on the functionality 💪🏿, Kindly make your content aligned to the center just like the wireframe 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Reservations](https://user-images.githubusercontent.com/69638013/181735584-df741c39-d4cb-4767-9d2e-901379d7262f.png)  |  ![Screenshot_363](https://user-images.githubusercontent.com/69638013/181735509-e24d7fc4-44b6-4eb3-beee-0780d960c612.png)